locales/en-US/formautofill.properties

```diff
@@ -3,26 +3,36 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 preferenceGroupTitle = Form Autofill
-enableProfileAutofill = Enable Profile Autofill
-savedProfiles = Saved Profiles…
+enableAddressAutofill = Autofill addresses
+learnMore = Learn more
+savedAddresses = Saved Addresses…
```

Comment:
What's the context here? Without any help, it either looks like a message to say that you saved addresses, and no ellipsis are needed, or "Saving Addresses…".

If it's a button label to access saved addresses, you should use a more intuitive ID (e.g. savedAddressesBtnLabel). That would also prevent developers from reusing the same string in a completely different context.

Reply:
`savedAddressesBtnLabel` sounds good!